src/lm_polygraph/utils/processor.py

```diff
@@ -76,13 +77,21 @@ def on_batch(
             print(f"{key}: {val}")
             print()
 
-    def on_eval(self, metrics: Dict[Tuple[str, str, str, str], float]):
+    def on_eval(
+        self,
+        metrics: Dict[Tuple[str, str, str, str], float],
+        bad_estimators: Dict[Estimator, int],
+    ):
         """
-        Outputs statistics from `metrics` to stdout.
+        Outputs statistics from `metrics` and failed estimators to stdout.
         """
         print("=" * 50 + " METRICS " + "=" * 50)
         print("Metrics:")
         print()
         for key, val in metrics.items():
             print(f"{key}: {val}")
             print()
+        if len(bad_estimators) > 0:
+            print("=" * 45 + " FAILED ESTIMATORS " + "=" * 45)
+            for bad_estimator, batch_i in bad_estimators.items():
+                print(str(bad_estimator), " on batch ", batch_i)
```

Comment:
On finished eval dump info about all failed estimators and batch id on which they failed.